### DISCUSSION TITLE
Urgent Action Required: Resolution Issues with mbp.hanabank.com due to AdGuard

### DISCUSSION THREAD

**Discussion by mt0205** [List-KR/community#14] [Q&A]
Dear [Recipient's Name],

I'm writing to bring to your attention a serious issue that has recently arisen due to AdGuard. The domain, mbp.hanabank.com, a critical part of a popular commercial banking app in Korea, is unable to resolve correctly. As a result, this has rendered the app non-functional, causing significant inconvenience to its numerous users.

I kindly request you to urgently look into this matter and ensure the proper resolution of the domain, mbp.hanabank.com, to restore functionality of the banking app.

Thank you in advance for your prompt attention to this matter.

Sincerely,

**Comment by piquark6046** [comment]
Does selecting AdGuard DNS server cause your issue?

**Reply by mt0205** [reply]
when I  nslookup for the 156.146.34.244 IP address you taught me, a timeout occurs. Please check if there is a typo in the IP address.

<img width="596" alt="image" src="https://github.com/List-KR/List-KR/assets/33386734/5b93f8b5-63f4-4b81-b6b5-70e982af5338">

**Reply by piquark6046** [reply]
Please refer https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291.

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291 를 참고해주세요.
